api/README.md

```diff
@@ -3,3 +3,10 @@
 ## JSON format
 
 JSON fields should be snake_case.
+
+## Documentation
+
+General guidelines, though not applicable in every case should be:
+
+- Request/Response model naming should follow: _[Subject][SubjectAction][Request|Response]_; I.E. **SessionListRequest, SessionListResponse, PasswordChangeRequest, PasswordResetResponse**...  
+Ideally this should add consistency to names, a benefit of grouping common Subjects and their Actions thus providing discoverability and readability boost. 
```

Comment:
Another question:
- Currently golang models are named - `_[Subject][SubjectAction][Request|Response]_`
- Swagger names has different names (but we add  just `..DTO` suffix)  - _[Subject][SubjectAction][Request|Response]DTO_
```
// IdentityRegistrationRequestDTO represents the identity registration user input parameters
// swagger:model IdentityRegistrationRequestDTO
type IdentityRegistrationRequest struct {
	// Stake is used by Provider, default 0
	Stake *big.Int `json:"stake,omitempty"`
	// Cache out address for Provider
	Beneficiary string `json:"beneficiary,omitempty"`
	// Fee: negotiated fee with transactor
	Fee *big.Int `json:"fee,omitempty"`
}
```

Reply:
Well - I don't know why `DTO` is added to the end in this case. Seems like noise. I would suggest 1:1 naming between Swagger documentation and actual code.